pkg/rancher-desktop/backend/lima.ts

```diff
@@ -423,16 +425,37 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
 
     const images = currentConfig.images.map(i => path.basename(i.location));
     // We had a typo in the name of the image; it was "alpline" instead of "alpine".
-    const versionMatch = images.map(i => /^alpl?ine-lima-v([0-9.]+)-/.exec(i)).find(defined);
+    // Image version may have a '+rd1' (or '.rd1') suffix after the upstream semver version.
+    const versionMatch = images.map(i => /^alpl?ine-lima-v([0-9.]+)(?:[+.]rd(\d+))?-/.exec(i)).find(defined);
     const existingVersion = semver.coerce(versionMatch ? versionMatch[1] : null);
+    const existingRDVersion = versionMatch ? versionMatch[2] : undefined;
 
     if (!existingVersion) {
       console.log(`Could not find base image version from ${ images }; skipping update of base images.`);
 
       return;
     }
 
-    const versionComparison = semver.coerce(IMAGE_VERSION)?.compare(existingVersion);
+    let versionComparison = semver.coerce(IMAGE_VERSION)?.compare(existingVersion);
+
+    // Compare RD patch versions if upstream semver are matching
```

Comment:
This chunk of code is complicated enough that a function (ideally with a unit test) might be worthwhile.

Reply:
I agree, but I don't want to do this right now; maybe in a separate PR